src/csharp/Microsoft.Spark.E2ETest/UdfTests/UdfComplexTypesTests.cs

```diff
@@ -219,7 +219,7 @@ public void TestUdfWithReturnAsRowType()
                 }
             }
 
-            // Generic row is a part of top-level column.
+            // GenericRow is a part of top-level column.

```

Comment:
What's the difference b/w this test and the one right above?

Reply:
Select udf returned column and another column to test top-level column. I guess we can remove this. What do you think?

Reply:
I think this was related to a bug we had before when we had one column in the `Row` object.
Can we change the comments to
```C#
// Test UDF that returns a Row object with multiple columns
```
and
```C#
// Test UDF that returns a Row object with a single column
```
?

Reply:
Sounds good. Comments changed.